### PR TITLE
Added method to check if jwt is valid.

### PR DESCRIPTION
**Pull request by tizianasellitto**
Added method is_jwt_valid in EmotlyService to check if jwt is valid.

### CODE REVIEW

**Comment by ggc87** [comment]
LGTM

File: Emotly/EmotlyService.swift [hunk 1]
```diff
@@ -351,13 +373,12 @@ class EmotlyService {
...
             self.jwt = jwt
         }
-
         return self.jwt
     }
 
     /// Used to get prepopulated HTTP headers (incl. the auth info).
     private  func reqHeaders() -> [String : String] {
-        return ["X-Emotly-Auth-Token" : jwt!.uglyString]
+        return ["X-Emotly-Auth-Token" : getJWT()!.uglyString]
```
Comment (michelangelo) [inline Emotly/EmotlyService.swift]: Would it be ok to throw if getJWT returns nil, here?

**Comment by michelangelo** [comment]
LGTM, just a remark on how to deal with possible `nil` JWT in reqHeaders.

File: Emotly/EmotlyService.swift (hunk 1 above)
Comment (tizianasellitto) [inline Emotly/EmotlyService.swift]: I think it should be user care to be sure that jwt is not nil when calling this.

**Pull request merged**